client/src/components/header/header.css

```diff
@@ -0,0 +1,24 @@
+body{
+    margin: 0;
+}
+p{
+margin: 0;
+display: flex;
+justify-content: center;
+align-items: center;
+
+}
+.header{
+    font-size: 24px;
```

Comment:
How about rem instead of px?  24px is 1.5rem.  Works for just about everything: fonts, margin, etc

Reply:
I agree,changed.